content/en/security_platform/cspm/findings.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Findings Explorer
+kind: documentation
+aliases:
+  - /security_platform/findings
+further_reading:
+- link: "security_platform/default_rules"
+  tag: "Documentation"
+  text: "Explore default cloud configuration rules"
+- link: "security_platform/cspm/frameworks"
+  tag: "Documentation"
+  text: "Learn about frameworks and industry benchmarks"
+---
+
+{{< site-region region="us" >}}
+<div class="alert alert-warning">
+Cloud Security Posture Management is currently in public beta.
+</div>
+{{< /site-region >}}
+
+{{< site-region region="us3,gov,eu" >}}
+<div class="alert alert-warning">
+Cloud Security Posture Management is not currently available in US1-FED or US3.
+</div>
+{{< /site-region >}}
+
+## Overview
+
+The Findings page helps drill into the details of CSPM findings, the configuration of resources, and descriptions and remediation guidelines for remediating a misconfigured resource.
```

Comment:
The Findings Explorer is dedicated to exploring the details of your security posture, including the configuration of individual resources and how they conform to rules, at any point in the past.

Use the time selector to explore your findings at any point in the past.